dangerfile.js

```diff
@@ -0,0 +1,133 @@
+"use strict";
```

Comment:
@VinulaUthsara please, remove the transpiled code and add it to `.gitignore` 👍

Reply:
dangerfile.js file removed, but causes a "deploy to preview" check to fail.